### PR TITLE
chore(llc,core,ui): update pubspec and changelogs

### PR DESCRIPTION
**Pull request by imtoori**

### CODE REVIEW

File: packages/stream_chat/lib/src/ws/websocket.dart [hunk 1]
```diff
@@ -112,6 +112,7 @@ class WebSocket with TimerHelper {
     if (_webSocketChannel != null) {
       _closeWebSocketChannel();
     }
+    // throw Error();
```
Comment (sachaarbonel) [inline packages/stream_chat/lib/src/ws/websocket.dart:115]: Maybe we can remove this commented code

**Review by sachaarbonel — CHANGES_REQUESTED** [review]
Left one comment otherwise looks good to me

File: packages/stream_chat/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## 3.5.1
+
+🐞 Fixed
+- Unread count specific to channel setting global unread count on a specific case.
+- The reconnection logic for the ws is now more robust.
```
Comment (HayesGordon) [inline packages/stream_chat/CHANGELOG.md:5]:
```suggestion
- The reconnection logic for the WebSocket connection is now more robust.
```

File: packages/stream_chat/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+## 3.5.1
+
+🐞 Fixed
+- Unread count specific to channel setting global unread count on a specific case.
```
Comment (HayesGordon) [inline packages/stream_chat/CHANGELOG.md:4]: This sentence is a bit unclear. Not too sure what this fixes

File: packages/stream_chat_flutter/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
+## 3.5.1
+
+🐞 Fixed
+
+- Mentions overlay now doesn't overflow when not enough height available
```
Comment (HayesGordon) [inline packages/stream_chat_flutter/CHANGELOG.md:5]:
```suggestion
- Mentions overlay now doesn't overflow when there is not enough height available
```

**Review by HayesGordon — APPROVED** [review]
LGTM, made some suggestions and comments on the Changelogs

File: packages/stream_chat/CHANGELOG.md (hunk 3 above)
Comment (imtoori) [reply packages/stream_chat/CHANGELOG.md:4]:
```suggestion
- `channel.unreadCount` was being set as using global unread count on a very specific case.
```

**Review by HayesGordon — DISMISSED** [review]
LGTM

**Pull request merged**